packages/orchestration/src/orchestration-types.ts

```diff
@@ -151,3 +152,28 @@ export interface DocumentGroundingServiceConfig {
    */
   output_param: string;
 }
+
+/**
+ * A descriptive  type for AzureThreshold input.
+ * @internal
+ */
+export interface AzureContentSafetyThresholdType {
+  /** Only safe content is allowed. AzureThreshold value of 0. */
+  readonly ALLOW_SAFE: AzureThreshold;
+  /** Safe and low-risk content is allowed. AzureThreshold value of 2.*/
+  readonly ALLOW_SAFE_LOW: AzureThreshold;
+  /** Safe, low-risk, and medium-risk content is allowed. AzureThreshold value of 4. */
+  readonly ALLOW_SAFE_LOW_MEDIUM: AzureThreshold;
+  /** All content is allowed. AzureThreshold value of 6. */
+  readonly ALLOW_ALL: AzureThreshold;
+}
+
+/**
+ * A descriptive constant for AzureThreshold.
+ */
+export const AzureContentSafetyThreshold: AzureContentSafetyThresholdType = {
```

Comment:
This is also another option:

```TS
export type AzureContentSafetyThresholdKeys =
  | 'ALLOW_SAFE'
  | 'ALLOW_SAFE_LOW'
  | 'ALLOW_SAFE_LOW_MEDIUM'
  | 'ALLOW_ALL';

export const AzureContentSafetyThreshold: Record<
  AzureContentSafetyThresholdKeys,
  AzureThreshold
> = {
  ALLOW_SAFE: 0,
  ALLOW_SAFE_LOW: 2,
  ALLOW_SAFE_LOW_MEDIUM: 4,
  ALLOW_ALL: 6
};
```

But then autocomplete looks like this (It doesn't give the value hints, even if I add comments):
<img width="1169" alt="Screenshot 2025-01-17 at 11 19 50" src="https://github.com/user-attachments/assets/873eaf3d-d8a5-4bd5-b977-898b372280db" />